### ISSUE TITLE
New datasets split

### ISSUE THREAD

**Issue by VSunN** [shahroudy/NTURGB-D#25]
Hi,
After downloading the new extended dataset, I found that the previous 60 classes of subjects also made new 60 class actions, but the new subject did not do the previous 60 classes, so that when the dataset is divided(eg, cross-setup eval), is there any gap when making predictions for new subjects?

**Comment by ljchangyu** [comment]
Thank you for this message. In NTU RGB+D 120 dataset, not all actions are performed by all subjects. This is because we design our dataset to cope with more realistic scenarios of human activity analysis. 

It is worth mentioning that the sample number of each class in NTU RGB+D 120 is still almost equal (~1000 samples for each action class).

**Comment by VSunN** [comment]
Shouldn't all subjects execute all classes? Under the cross-setup division, the new subjects in the training set are not in the previous category, so the test will not be predicted as the previous category.

**Comment by ljchangyu** [comment]
Kindly refer to our paper, Section 3.2.2:

For cross-setup evaluation, we pick all the samples with **even collection setup IDs** for training, and those with **odd setup IDs** for testing, i.e., 16 setups are used for training, and the other 16 setups are reserved for testing.

With this setting, all action classes will appear in both training set and testing set.

**Comment by VSunN** [comment]
Thanks for your kindly reply,
I know that there are 120 categories in the training set and the test set, but the problem is that not all subjects in the training set have 120 categories (about half of the subjects are only 60 categories?), will this introduce some prior knowledge, these subjects in the testing set will only predict 60 of the 120 categories too.

**Comment by ljchangyu** [comment]
The obtained samples of some subjects cover all actions, while samples of some other subjects cover a part of the actions.  The datasets are collected across a long period to get enough samples, and are built as two major versions.  It is difficult to ask all the previous subjects to perform new actions. But this is also the realistic scenario for activity analysis.
